pkg/cmd/policy_enable.go

```diff
@@ -48,15 +50,32 @@ func newPolicyEnableCmd() *cobra.Command {
 				version = &cliArgs[1]
 			}
 
+			// Load the configuration from the user-specified JSON file into config object.
+			var config map[string]*json.RawMessage
+			if args.configFile != "" {
+				config, err = loadJSONConfigFile(args.configFile)
+				if err != nil {
+					return err
+				}
+			}
+
 			// Attempt to enable the Policy Pack.
-			return policyPack.Enable(commandContext(), args.policyGroup, backend.PolicyPackOperation{
-				VersionTag: version, Scopes: cancellationScopes})
+			return policyPack.Enable(commandContext(), args.policyGroup,
+				backend.PolicyPackOperation{
+					VersionTag: version,
+					Scopes:     cancellationScopes,
+					Config:     config,
+				})
 		}),
 	}
 
 	cmd.PersistentFlags().StringVar(
 		&args.policyGroup, "policy-group", "",
 		"The Policy Group for which the Policy Pack will be enabled; if not specified, the default Policy Group is used")
 
+	cmd.PersistentFlags().StringVar(
+		&args.configFile, "config-file", "",
```

Comment:
I think when running a policy-pack locally there is a different flag name used for the config file. Should we use the same here for consistency?

Reply:
For `pulumi preview/up/watch`, the flag is `--policy-pack-config`. Since this command is already about policies, I think we can leave off the `policy-pack` prefix, and just use `--config`.